tests/tests_pytorch/graveyard/test_cloud_io.py

```diff
@@ -0,0 +1,69 @@
+import glob
+import os
+import sys
+import warnings
+
+import pytest
+import torch
+
+import pytorch_lightning  # noqa: F401
+from tests_pytorch import _PATH_LEGACY, _PROJECT_ROOT
+
+LEGACY_CHECKPOINTS_PATH = os.path.join(_PATH_LEGACY, "checkpoints")
+CHECKPOINT_EXTENSION = ".ckpt"
+# load list of all back compatible versions
+with open(os.path.join(_PROJECT_ROOT, "legacy", "back-compatible-versions.txt")) as fp:
+    LEGACY_BACK_COMPATIBLE_PL_VERSIONS = [ln.strip() for ln in fp.readlines()]
+
+
+@pytest.mark.parametrize("pl_version", LEGACY_BACK_COMPATIBLE_PL_VERSIONS)
+@pytest.mark.skipif(
+    not "pytorch_" + "lightning" in sys.modules, reason="This test is only relevant for the standalone package"
```

Comment:
the string addition can be collapsed in a single string, right?

Reply:
Yes and no, we want to preserve this even in the mono-package and the setup is running a naive replacement, so to prevent it we need to be creative...

Reply:
It could, but then this would be replaced when converting the tests for the unified mirror package which we don't want. So this is kind of hacky to prevent this